bpf/bpf_lxc.c

```diff
@@ -371,8 +378,11 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 		set_encrypt_dip(ctx, tunnel_endpoint);
 #endif
 	} else
-#endif
-#endif
+#elif defined(ENABLE_WIREGUARD)
+	if (dst_remote_ep)
+		set_encrypt_mark(ctx);
```

Comment:
Seems like this also needs the } else to make sure that the mark being set here doesn't immediately get overwritten..?

Or even if ENABLE_WIREGUARD and ENABLE_IDENTITY_MARK are mutually exclusive, that's not clear when reading this code right here.